source/infrastructure/instance-scheduler/lib/event-bus-stack.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import * as events from "aws-cdk-lib/aws-events";
+import * as cdk from "aws-cdk-lib";
+import { Construct } from "constructs";
+
+export interface SchedulerEventBusProps {
+  organizationId: string[];
+  namespace: string;
+  lambdaFunctionArn: string;
+  eventBusName: string;
+  isMemberOfOrganizationsCondition: cdk.CfnCondition;
+}
+
+export class SchedulerEventBusResources extends Construct {
```

Comment:
should this be a capital C Construct or just a helper class like I did for core-scheduler.ts? @hearde your thoughts?